pkg/sqlite/migrationsPostgres/1_initial.up.sql

```diff
@@ -0,0 +1,475 @@
+CREATE COLLATION IF NOT EXISTS NATURAL_CI (provider = icu, locale = 'en@colNumeric=yes');
```

Comment:
One note that this requires Postgres to be compiled with ICU support, which I believe is optional. Could this be made optional maybe?

Reply:
You wouldn't have me... write MORE C LANG?! 😱😱😱
Yeah maybe i could, i'll have to look at it

Reply:
LOL I would not suggest to put more C

I was just thinking if there was a way to make the natural collation optional. If it's not available in the server, then just fallback to a "regular" sorting (non-natural)?

Reply:
Postgresql 16 bundles ICU by default, so im gonna let it be a hard dependency